labot/protocol.py

```diff
@@ -54,6 +55,13 @@ def read(type, data):
             ans[var['name']] = readVec(var, data)
         else:
             ans[var['name']] = read(var['type'], data)
+    if type['name'] == "NetworkDataContainerMessage":
```

Comment:
I don't think this change belongs to this PR. Could you revert the concerned commit in the PR branch, or start a new PR?